src/arviz_plots/plots/ppctstat.py

```diff
@@ -0,0 +1,194 @@
+        plot_kwargs = {}
+    else:
+        plot_kwargs = plot_kwargs.copy()
+    if pc_kwargs is None:
+        pc_kwargs = {}
+    else:
+        pc_kwargs = pc_kwargs.copy()
+    if labeller is None:
+        labeller = BaseLabeller()
+
+    if backend is None:
+        if plot_collection is None:
+            backend = rcParams["plot.backend"]
+        else:
+            backend = plot_collection.backend
+
+    plot_bknd = import_module(f".backend.{backend}", package="arviz_plots")
+    if aes_map is None:
+        aes_map = {}
+    else:
+        aes_map = aes_map.copy()
+
+    if data_pairs is None:
+        data_pairs = (var_names, var_names)
+    else:
+        data_pairs = (list(data_pairs.keys()), list(data_pairs.values()))
+
+    if group == "posterior":
+        predictive_dist = process_group_variables_coords(
+            dt, group="posterior_predictive", var_names=data_pairs[0], filter_vars=filter_vars, coords=coords
+        )
+    elif group == "prior":
+        predictive_dist = process_group_variables_coords(
+            dt, group="prior_predictive", var_names=data_pairs[0], filter_vars=filter_vars, coords=coords
+        )
+
+    predictive_dist = predictive_dist.stack(sample=sample_dims)
+    if t_stat == "median":
+        predictive_dist = predictive_dist.median(dim=list(predictive_dist.dims)[0])
+    elif t_stat == "mean":
+        predictive_dist = predictive_dist.mean(dim=list(predictive_dist.dims)[0])
+    elif t_stat == "std":
+        predictive_dist = predictive_dist.std(dim=list(predictive_dist.dims)[0])
+    else:
+        try:
+            t_stat_float = float(t_stat)
+        except ValueError:
+            raise ValueError(f"T statistics '{t_stat}' not implemented")
+        if 0 < t_stat_float < 1:
+            return predictive_dist.quantile(q=t_stat_float, dim=list(predictive_dist.dims)[0])
+        else:
+            raise ValueError(f"T statistic '{t_stat}' not in valid range (0, 1).")
+    if plot_collection is None:
+        pc_kwargs["plot_grid_kws"] = pc_kwargs.get("plot_grid_kws", {}).copy()
+
+        pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+        pc_kwargs["aes"].setdefault("overlay", ["sample"])
+        pc_kwargs.setdefault("cols", "__variable__")
+        pc_kwargs.setdefault("rows", None)
+
+        pc_kwargs = set_figure_layout(pc_kwargs, plot_bknd, predictive_dist)
+
+        plot_collection = PlotCollection.wrap(
+            predictive_dist,
+            backend=backend,
+            **pc_kwargs,
+        )
+    # Plot KDE lines
+    kde_kwargs = copy(plot_kwargs.get("kde_line", {}))
+    kde_dt = predictive_dist.azstats.kde(dims=["sample"], **stats_kwargs)
+    if kde_kwargs is not False:
+        _, _, kde_ignore = filter_aes(plot_collection, aes_map, "kde_line", sample_dims)
+
+        plot_collection.map(
+            line_xy,
+            "kde_line",
+            data=kde_dt,
+            ignore_aes=kde_ignore,
+            **kde_kwargs
+        )
```

Comment:
better to call `plot_dist`, by doing so we get "for free" more alternatives, KDE, ecdf, histogram (and in the future, other representations like quantile dot plots, etc).

Reply:
Sounds good, have made the change